ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.js

```diff
@@ -2,8 +2,11 @@ import React, { useCallback, useContext, useEffect, useMemo } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { useHistory } from 'react-router-dom';
 import { ethErrors, serializeError } from 'eth-rpc-errors';
-import ActionableMessage from '../../components/ui/actionable-message/actionable-message';
-import Button from '../../components/ui/button';
+import {
+  BUTTON_VARIANT,
```

Comment:
I believe `BUTTON_VARIANT` has now been deprecated in favor of the `ButtonVariant` enum. Could you please replace?

Reply:
Ok, replaced the enum.

Reply:
Currently, I am facing this bug #19818, as a result I couldn't update the screenshot. Any suggestion on how to avoid this bug ? I am on Ubuntu.

Reply:
Hey @kumavis, would it be possible to support @pritam1813 on his lavamoat issue?